contracts/adapters/curve/CurveV1_Base.sol

```diff
@@ -424,71 +514,102 @@ contract CurveV1AdapterBase is AbstractAdapter, ICurveV1Adapter {
     /// @dev Internal implementation of `remove_liquidity`
     ///      - passes calldata to the target contract
     ///      - enables all pool tokens
-    function _remove_liquidity() internal {
+    function _remove_liquidity() internal returns (uint256 tokensToEnable, uint256 tokensToDisable) {
         _execute(msg.data);
-        _changeEnabledTokens(token0Mask | token1Mask | token2Mask | token3Mask, 0); // F: [ACV1_2-5, ACV1_3-5, ACV1_4-5]
+        (tokensToEnable, tokensToDisable) = (token0Mask | token1Mask | token2Mask | token3Mask, 0); // F: [ACV1_2-5, ACV1_3-5, ACV1_4-5]
```

Comment:
BitMask is also preferale

Reply:
`BitMask` didn't exist when these changes were made :D

I'll fix in the optimization/cleanup PR, this one is purely to fix compatibility